.env

```diff
@@ -1,4 +1,4 @@
-DOCKER_TAG=v1.12.0
+DOCKER_TAG=v1.13.0
```

Comment:
Did you mean to make this 1.13 instead of 1.14?

Reply:
Nope; I was just focusing on 1.13.0 on this branch and have 1.14.0 updates on a separate branch.

Reply:
Ah ok. I saw two PR's come in and must have gotten them confused. 👍